appinfo/routes.php

```diff
@@ -648,6 +648,30 @@
 				'apiVersion' => 'v1',
 			],
 		],
+		[
+			'name' => 'Settings#blockUser',
+			'url' => '/api/{apiVersion}/settings/block-user',
+			'verb' => 'POST',
+			'requirements' => [
+				'apiVersion' => 'v(4)'
```

Comment:
```suggestion
				'apiVersion' => 'v2'
```

Settings is on v1 at the moment, so we can go with v2?